templates/mixplank/next/src/app/data.ts

```diff
@@ -1,18 +1,21 @@
 import { createClient } from "@clickhouse/client-web"
 
-const client = createClient({
+export const clickhouseClient = createClient({
     host: "http://localhost:18123",
     username: "panda",
     password: "pandapass",
     database: "local",
```

Comment:
not sure if is not already on your radar but making this configurable through env will be needed.